### ISSUE TITLE
Problem with coordinates of boxes

### ISSUE THREAD

**Issue by mohammadkamalzare** [Microsoft/VoTT#350]
Recently, I used VoTT to create a new dataset, and I used pascal voc export format. Now, I realize that the coordinates of boxes in some of .xml files have problems and they are more than their corresponding image size. For example, There is an image with width of 261px and height of 193, and the xmin of a sample box in the image is 424!!!. I do not know how this might happen! please help me. Thanks.

**Comment by kichinsky** [comment]
1. Are all the images of the same size? 

2. Can you check the json-file generated by VoTT for your dataset? What is stored for your image?

**Comment by mohammadkamalzare** [comment]
No they are not. The images size and format are varied.
The json-files also have been correctly stored, and I use them to create xml-files by myself. The problem is that I did not save some of the json-files.

**Comment by kichinsky** [comment]
Yeah, we noted this issue in #345. Sorry, for your loss, VoTT wasn't designed to support that use case, but we are looking into how to fix that issue.

Not sure if it will help you to recreate the data, but the way how it works right now for images is that it uses the width/height of the first frame for the rest of calculations while exporting. So maybe the coordinates in xml-file are ok, but the image width/height is wrong?

**Comment by mohammadkamalzare** [comment]
Fortunately, I found the problem early, and the problematic xml files are not much. So, it's okay.

I checked the image width/height of 3 xml files but their width and height were correct. Perhaps, the coordinates are calculated based on first image size; I mean, perhaps, they are normalized at the time based on current image size, and then they are recalculated based on first image width and height while exporting!?

**Comment by Legor** [comment]
It seems to me that this has been adressed allready in #283. The solution there seems to store the image size for every single annotation, which to me seems a bit of an overkill. It should be enough to store the image size just once. 

I also did a quick test with images of different size and the annotated boxes could be restored the right way for all images.

**Comment by mohammadkamalzare** [comment]
It seems that the solution of #283 works for me.
@Legor Thanks a lot.
